lite/examples/pose_estimation/android/app/src/androidTest/java/org/tensorflow/lite/examples/poseestimation/EvaluationUtils.kt

```diff
@@ -8,21 +24,21 @@ import com.google.common.truth.Truth.assertThat
 import com.google.common.truth.Truth.assertWithMessage
 import org.tensorflow.lite.examples.poseestimation.data.BodyPart
 import org.tensorflow.lite.examples.poseestimation.data.Person
+import java.io.BufferedReader
+import java.io.InputStreamReader
 import kotlin.math.pow
 
 object EvaluationUtils {
 
-    private const val ACCEPTABLE_ERROR = 10f // max 10 pixels
-    private const val BITMAP_FIXED_WIDTH_SIZE = 400
-
     /**
      * Assert whether the detected person from the image match with the expected result.
      * Detection result is accepted as correct if it is within the ACCEPTABLE_ERROR range from the
      * expected result.
      */
     fun assertPoseDetectionResult(
         person: Person,
-        expectedResult: Map<BodyPart, PointF>
+        expectedResult: Map<BodyPart, PointF>,
+        acceptableError: Float
```

Comment:
For kotlin, you can add default value: `acceptableError: Float = ACCEPTABLE_ERROR`

Reply:
I changed the test code to compare each model output to a ground truth value, instead of comparing each model's output to its own golden output value.

As each model has different level of accuracy, I set different acceptable error value for each model test (e.g. MoveNet Thunder is more accurate so acceptable error is small, while PoseNet is less accurate so its value is high).

Reply:
acceptableError differs between models so we can't set a default value. Instead, each test method needs to pass in the value specific to each model (e.g. MoveNet Thunder, MoveNet Lightning, PoseNet)